playground/Elasticsearch/Elasticsearch.ApiService/Program.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Elasticsearch.Net;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+
+builder.Services.AddSingleton<ElasticLowLevelClient>((sp) =>
+{
+    var settings = new ConnectionConfiguration(new Uri(builder.Configuration.GetConnectionString("elasticsearch")!))
+    .RequestTimeout(TimeSpan.FromMinutes(2));
+
+    var lowlevelClient = new ElasticLowLevelClient(settings);
+    return lowlevelClient;
+});
+
+var app = builder.Build();
+
+app.MapGet("/", async (ElasticLowLevelClient elasticClient) =>
+{
+    var response = await elasticClient.GetAsync<StringResponse>("people", "1");
+    return response.Body;
+});
+
+app.MapPost("/", async (ElasticLowLevelClient elasticClient) =>
```

Comment:
It's easier if these are all `Get`s, so you can use them without a dedicated HTTP endpoint tool. See 

https://github.com/dotnet/aspire/blob/87570cc97aa38e24f4eb4e1eb85b6926fd470025/playground/Qdrant/Qdrant.ApiService/Program.cs#L19

For how this is done elsewhere.